lmdb-sys/build.rs

```diff
@@ -16,6 +16,20 @@ fn main() {
         if target.contains("android") {
             build.define("ANDROID", "1");
         }
+        if target.contains("windows") {
+            // LMDB on Windows has an issue with ´off_t´ being defined as ´long´ 32-bit signed
+            // which caused a max size of the database of 2 GB which we work 
+            // around by redefining ´off_t´ to 64-bit
+            // 
+            // was discussed here and allgedly fixed in January 2016 but fix doesn't work as 
+            // Windows doesn't support _FILE_OFFSET_BITS
+            // https://www.openldap.org/lists/openldap-bugs/201605/msg00015.htm
```

Comment:
This URL returns a 404 NOT FOUND error. It looks like the correct URL is https://www.openldap.org/lists/openldap-bugs/201605/msg00015.html (with the trailing "l"; perhaps due to a recent reconfiguration of the server?).